third_party/efr32_sdk/.clang-format

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
Why is this needed here?

Reply:
Added to maintain consistency while porting files from other internal sources into MATTER.